Sources/TuistKit/Graph/GraphNode.swift

```diff
@@ -131,6 +134,49 @@ enum PrecompiledNodeError: FatalError, Equatable {
     }
 }
 
+class SDKNode: GraphNode {
+    enum Error: Swift.Error {
```

Comment:
Errors should be of type `FatalError` *(sorry for not having this one documented yet)*. You can check the type [here](https://github.com/tuist/tuist/blob/master/Sources/TuistCore/Errors/FatalError.swift). Notice that a `FatalError` contains information about the type of error being raised:
- Abort: The execution cannot continue, in most cases because the user is misusing the tool or something in the environment is misconfigured.
- Bug: There's a bug and we need it to be reported.

It also contains a description which is output to the user. It's important that this one is very descriptive about what happened.